test/object-store/schema.cpp

```diff
@@ -410,18 +410,6 @@ TEST_CASE("Schema") {
                     "Property 'object.link' of type 'object' cannot be a link to an asymmetric object."));
         }
 
-        SECTION("rejects link properties with asymmetric origin object") {
```

Comment:
Rather than removing the test entirely the test should be changed to verify that it's allowed.

Reply:
Fixed

Reply:
FIxed